<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="Setup_Tab3_SpeciesGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Species</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_85">
        <item>
         <widget class="QLabel" name="Setup_Tab3_NumSpeciesLBL">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Number of species in the table</string>
          </property>
          <property name="statusTip">
           <string>Number of species in the table</string>
          </property>
          <property name="text">
           <string>Num Species:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Setup_Tab3_NumSpeciesSB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Number of species in the table</string>
          </property>
          <property name="statusTip">
           <string>Number of species in the table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Setup_Tab3_AddSpeciesPB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Add a new Species after the last Species.</string>
          </property>
          <property name="statusTip">
           <string>Add a new Species after the last Species.</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Setup_Tab3_DelSpeciesPB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Delete the currently selected Species</string>
          </property>
          <property name="statusTip">
           <string>Delete the currently selected Species</string>
          </property>
          <property name="text">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Setup_Tab3_ReloadSpeciesPB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Reload current table from the saved database. </string>
          </property>
          <property name="statusTip">
           <string>Reload current table from the saved database. </string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Reload&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Reload the below Species table with what's currently saved in the database. Please note that all data in the Species table will be replaced by what's in the database.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_91">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="Setup_Tab3_SpeciesTW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>This table defines the Species in the current database table.</string>
        </property>
        <property name="statusTip">
         <string>This table defines the Species in the current database table.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_87">
     <item>
      <widget class="QPushButton" name="Setup_Tab3_PrevPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to the previous page</string>
       </property>
       <property name="statusTip">
        <string>Go to the previous page</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_95">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Setup_Tab3_LoadPB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Load data for above table</string>
       </property>
       <property name="statusTip">
        <string>Load data for above table</string>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Setup_Tab3_SavePB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Save data from above table</string>
       </property>
       <property name="statusTip">
        <string>Save data from above table</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_42">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Setup_Tab3_NextPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to the next page</string>
       </property>
       <property name="statusTip">
        <string>Go to the next page</string>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
